src/developer_reference.md

```diff
@@ -4,3 +4,63 @@
 developers. Describes the layout of modules, how to navigate the reference, the
 scope of the main nannou crate i.e. the kinds of work that should go inside the
 main repo and the kinds that might be better off in separate repos, etc.
+
+# Updating old code
+
+Sometimes you have to touch old code and make it work in a new environment.
+This sections helps you deal with problems you might encounter.
+
+In general it helps to read all the CHANGELOG files from your (old) source
+version to the target you want to upgrade to.
+
+
+## Sketches don't run
```

Comment:
Perhaps we should put the "Sketches don't run" and "with_dimension is gone" under a "Version 0.13 Changes" heading? It might help to clarify in the future that these changes are specific to version 0.13 e.g.

```
## Version 0.13 Changes

### Sketches don't run

...

## Window sizing does not work / with_dimension is gone

...
```

Reply:
Right now they are specific to 0.13, but I advise against it. Later people are going to look for their problems (in the headings) and not for version changes.

Reply:
I think the version number can be useful for checking which changes occurred during the period of time since the user updated their code. The sub-headings can still be there.